validation/standard/standard-versions-sepolia.toml

```diff
@@ -1,12 +1,10 @@
-[releases]
-
 # Contracts which are
 #   * unproxied singletons: specify a standard "address"
 #   * proxied             : specify a standard "implementation_address"
 #   * neither             : specify neither a standard "address" nor "implementation_address"
 
 # Holocene https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.8.0-rc.4
-[releases."op-contracts/v1.8.0-rc.4"]
+["op-contracts/v1.8.0-rc.4"]
```

Comment:
This is a breaking change, with respect to at least the Holocene upgrade scripts. I think we should still do it, but coordinate with this PR https://github.com/ethereum-optimism/optimism/pull/13412 to cope with the change.